model/translation.go

```diff
@@ -46,7 +46,10 @@ func (t *Translation) State() string {
 // NewTranslationFromRequest creates a new Translation from a
 // TranslationRequest
 func NewTranslationFromRequest(translationRequest *TranslationRequest) *Translation {
-	teamName := cleanTeamName(translationRequest.Team)
+	teamName := translationRequest.Team
+	if translationRequest.Type != MattermostWorkspaceBackupType {
+		teamName = cleanTeamName(teamName)
+	}
```

Comment:
Looks like we may want some interfaces in the future if we continue to add other import types to keep things clean.

Reply:
I think this will be the only type of import where this step is skipped @gabrieljackson so it probably doesn't make sense to generalize it

I'm not aware of any other offerings that have a "Teams" concept like Mattermost does. Any other imports would follow the same pattern as Slack imports, where a "Slack Workspace" maps to a single Team.

Reply:
Sorry, if you saw it before I deleted it, I made a comment that didn't make any sense

Yeah it's possible that we need another interface here, I'd have to think about it, but it'd definitely be a future improvement. 

I do think that this one-off check will only ever be for the Mattermost type, since a team name will have to be specified for any other import type. Only Mattermost import types will already include team names (and already in the approved format), so only Mattermost import types can skip the sanitization step & trust the incoming import.

If this starts to get complicated down the line we can add an interface.